iib/web/api_v1.py

```diff
@@ -917,3 +930,105 @@ def create_empty_index():
 
     flask.current_app.logger.debug('Successfully scheduled request %d', request.id)
     return flask.jsonify(request.to_json()), 201
+
+
+@api_v1.route('/builds/recursive-related-bundles', methods=['POST'])
+@login_required
+def recursive_related_bundles():
+    """
+    Submit a request to get nested related bundles of an operator bundle image in a DFS manner.
```

Comment:
hm, actually I implemented a level order traversal. So all child nodes will appear first in the list . And, I agree we should mention it here and somewhere in the code. On it.

Reply:
a reversed level-order traversal? I get lost in all of the possible variants... especially when you diverge from the "named ones" . If you write it, I am sure that it will be sufficient.